HW1/P5/P5.txt

```diff
@@ -0,0 +1,9 @@
+# Your discussion here
+
+Finding Directed Paths:
+From “Kevin Bacon” to “Harvard University”, one of the shortest path is 
+[u'Kevin_Bacon', u'Marisa_Silver', u'Harvard_University'](takes only a few seconds)
+
+For the opposite direction, it'll take too much time. 
+
```

Comment:
Calls to `collect` within your loop are the main cause of long compute times here.  Instead, consider keeping a running list of nodes you've already touched, which you can use to filter the main graph on each iteration.  Then, call `collect` only once when finished.